dev_docs/tutorials/performance/adding_performance_journey.mdx

```diff
@@ -42,18 +44,36 @@ export const journey = new Journey({
   });
 ```
 
+Alternative to archives is to use Synthtrace ES client:
+
+```
+export const journey = new Journey({
+  synthtrace: {
+    type: 'apm',
+    generator: generateApmData,
+    options: {
+      from: new Date(Date.now() - 1000 * 60 * 15),
+      to: new Date(Date.now() + 1000 * 60 * 15),
+    },
```

Comment:
nit but wondering if the options are necessary or if we can either bind arguments to the generator or have curried generators

Reply:
Good point! I don't have strong opinion, will push a change to make it optional.